plugin/color.py

```diff
@@ -19,11 +20,11 @@ def run(self, edit: sublime.Edit, color_information: ColorInformation) -> None:
         if session:
             self._version = self.view.change_count()
             self._range = color_information['range']
-            params = {
+            params = cast(ColorPresentationParams, {
```

Comment:
Is this fix for mypy? Pyright seems to do fine with it.

It's very much preferred to not cast things since that provides no type safety regarding the keys of the object.

Mypy just doesn't support NotRequired in some context and I don't want to makes types less safe just for it.

Reply:
> Pyright instead will ignore the actual expression and assume the specified type for it.

That's exactly what `cast` does and what we want to avoid. And also that's not how it works here with Pyright. Pyright also asserts the type and ensures that the properties match the type. For example:

![Screenshot 2023-01-05 at 10 31 31](https://user-images.githubusercontent.com/153197/210751108-6d2895b4-8ee8-41c7-ba7c-0abcb9e1d3c2.png)